resources/assets/lib/chat/chat-state-store.ts

```diff
@@ -157,6 +160,19 @@ export default class ChatStateStore implements DispatchListener {
     this.selectChannel(event.channelId);
   }
 
+  @action
+  private handleFriendUpdated(event: FriendUpdated) {
+    if (!this.isChatMounted) return;
+
+    // TODO: update other tabs...somehow
```

Comment:
doesn't relevant for non-pm channels? 🤔 what to update here

Reply:
Other tabs with chat open currently don't get updated with the new state